susemanager-utils/susemanager-sls/salt/packages/profileupdate.sls

```diff
@@ -1,6 +1,7 @@
 packages:
   module.run:
     - name: pkg.info_installed
+{% if grains['os_family'] == 'Suse' or grains['os_family'] == 'RedHat' %}
```

Comment:
Plan was to uniform `info_installed` so that it behaves the same in Debian too, and IIUC the Salt part is done:

https://github.com/SUSE/salt-board/issues/155

I then wonder if we should expand the `os_family` selection here, or even remove the condition altogether.

Reply:
Good catch @moio. I removed the condition.